From 39b04980cf33f6c9a2522fe64b5fcdb02e8b61ec Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:09:07 -0400
Subject: [powerpc] Add "force config cmd line" Kconfig option

Message-id: <1410545655-205645-319-git-send-email-dzickus@redhat.com>
Patchwork-id: 94685
O-Subject: [RHEL7 PATCH 318/626] powerpc: Add "force config cmd line" Kconfig option
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit eb3b80f676c8b610bcf5e3ba58d2876a04f74b16
Author: Sebastian Siewior <bigeasy@linutronix.de>
Date:   Thu Feb 20 21:48:17 2014 +0100

    powerpc: Add "force config cmd line" Kconfig option

    powerpc uses early_init_dt_scan_chosen() from common fdt code. By
    enabling this option, the common code can take the built in
    command line over the one that is comming from bootloader / DT.

    Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/Kconfig b/arch/powerpc/Kconfig
index ab693ec..ca7f90a 100644
--- a/arch/powerpc/Kconfig
+++ b/arch/powerpc/Kconfig
@@ -623,6 +623,15 @@ config CMDLINE
    some command-line options at build time by entering them here.  In
    most cases you will need to specify the root device here.
 
+config CMDLINE_FORCE
+ bool "Always use the default kernel command string"
+ depends on CMDLINE_BOOL
+ help
+   Always use the default kernel command string, even if the boot
+   loader passes other arguments to the kernel.
+   This is useful if you cannot or don't want to change the
+   command-line options your boot loader passes to the kernel.
+
 config EXTRA_TARGETS
  string "Additional default image types"
  help
-- 
1.7.1